run_kraken.py

```diff
@@ -88,17 +97,24 @@ def main(options, command: Optional[str]) -> int:
         iterations = get_yaml_item_value(config["tunings"], "iterations", 1)
         daemon_mode = get_yaml_item_value(config["tunings"], "daemon_mode", False)
 
-        prometheus_url = config["performance_monitoring"].get("prometheus_url")
-        prometheus_bearer_token = config["performance_monitoring"].get(
-            "prometheus_bearer_token"
-        )
+        # Prometheus configuration: environment variables take precedence over config file
```

Comment:
these variables are already set in the config by [these lines](https://github.com/krkn-chaos/krkn/blob/4527d073c6a69af86fcbc9f7c3be19b064d40e4e/run_kraken.py#L91-L94). Let's remove these